src/Schema/Values/Tab.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Master list of tabs available for the iOS app Tab Bar (footer navigation)
+ *
+ * @packageDocumentation
+ */
+
+export type Tab =
```

Comment:
The current set of values (as of 6.4.9) should be "home", "search", "messages", "sell" and "favorites" -- from left to right, on the tab bar. This is the existing naming of each tab that have been using on the event `eigen_production.profile_view`

Reply:
yep this covers our bases. We probably won't use `cityGuide` or `favorites` ever again but it might be good to keep them for historical context.